lib/reporters/dot_reporter.js

```diff
@@ -43,8 +46,10 @@ module.exports = class DotReporter {
       this.currentLineChars = 0;
       this.out.write('\n  ');
     }
-    if (result.passed) {
+    if (result.passed && !result.todo) {
       this.out.write('.');
+    } else if (!result.passed && result.todo) {
```

Comment:
can probably just have `else if(result.todo) {` here since `result.passed` would be false (goes towards the rest fo the changes in the PR)

Reply:
Per my previous comment, you can have a passing todo in QUnit. This is considered a bad thing, since you should mark it as a regular test, not a todo anymore.

Reply:
resolving this as part of the other comment